iso/Dockerfile

```diff
@@ -20,6 +20,10 @@ FROM boot2docker/boot2docker:1.12.3
 RUN echo "1.12.3" > $ROOTFS/etc/version
 ENV DOCKER_SHA a3bb88649bcc47775c29dab7d95709f4dda04d0e
 
+# Add the /etc/os-release file
+ADD os-release $ROOTFS/etc
+RUN sed -i '/os-release/,/^EOOS/d' /make_iso.sh
```

Comment:
I need to do this to remove the code https://github.com/boot2docker/boot2docker/blob/master/rootfs/make_iso.sh#L19-L34 otherwise it was getting overwritten by the script at the end of the Dockerfile